superset/assets/src/datasource/DatasourceEditor.jsx

```diff
@@ -211,7 +213,7 @@ export class DatasourceEditor extends React.PureComponent {
       databaseColumns: props.datasource.columns.filter(col => !col.expression),
       calculatedColumns: props.datasource.columns.filter(col => !!col.expression),
       metadataLoading: false,
-      activeTabKey: 1,
+      activeTabKey: 4,
```

Comment:
do you think we should actually reorder the tabs instead of just defaulting to the last one? not sure here tbh

Reply:
fixed.